.env.deploy

```diff
@@ -4,6 +4,7 @@ APP_KEY=
 APP_TIMEZONE=UTC
 
 DB_HOST=
+DB_HOST_REPLICA=
```

Comment:
`DB_HOST_REPLICA=""`

will make it an optional variable when processed by the deployment ansible playbooks